### PR TITLE
add html5 form

### PR DESCRIPTION
**Pull request by AnnaGrynchuk**
### HTML5  form

[Demo](https://annagrynchuk.github.io/Form-HTML5/) |
[Code base](https://github.com/AnnaGrynchuk/Form-HTML5)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/AnnGrynchuk/html5_form/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                    </div>
+                    <div class="card">
+                        <div class="number">
+                            <label class="label_number"  for="card_number">Card number</label>
+                            <input class="card_number" type="text"  inputmode="numeric" id="card_number" autocomplete="off" pattern="[0-9]*" minlength="16" maxlength="16" title="Enter your card number" name="card_number" value="" required>
+                            <span class="visa_icon"></span>
+                        </div>
+                        <div class="date">
+                            <label class="label_date"  for="date">Expiry date</label>
+                            <input class="card_date" type="text"  inputmode="numeric" id="date" autocomplete="off" placeholder="DD/MM/YY" pattern="\d{1,2}/\d{1,2}/\d{2}" title="Enter card expiry date" name="card_date" value=""  required>
```
Comment (yevhenorlov) [inline submissions/AnnGrynchuk/html5_form/index.html]: The format is not clear from the tip. Also, 99/99/99 shouldn't be a valid date.
I would recommend using an `<input type="date">` here

File: submissions/AnnGrynchuk/html5_form/index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                                <option value="kurdish">Kurdish</option>
+                                <option value="arabic">Arabic</option>
+                                <option value="english">English</option>
+                                <option value="spanish">Spanish</option>
+                                <option value="italian">Italian</option>
+                                <option value="ukrainian">Ukrainian</option>
+                                <option value="indian">Indian</option>
+                            </optgroup>
+                        </select>
+                        <span><input class="remove_lang" type="checkbox"></span>
```
[the remark this answers is not in the document]
Comment (AnnaGrynchuk) [reply submissions/AnnGrynchuk/html5_form/index.html]: Only one reason - I couldn't find way to stylize button like this checkbox with fontawesome icon) Button doesn't dissappear with  -webkit-appearance and I can't hide it even if I change it's size to minimal. It works only if I set backround-color: white; and border:none; to button, but I'm not sure that it's a good solution.  Can you help me?

[the remark this answers is not in the document]
Comment (yevhenorlov) [reply submissions/AnnGrynchuk/html5_form/index.html]: I'd say that 
```css
border: none;
background: transparent;
```
would be a pretty good solution for button styling

**Comment by AnnaGrynchuk** [comment]
@yevhenorlov , thank you for review! I've tried to fix all things.

**Pull request merged**